evergreen/evergreen.yml

```diff
@@ -236,10 +251,49 @@ functions:
         working_dir: mongo-aspnetcore-odata
         env:
           NUGET_KEY: ${nuget_key}
+          NUGET_SIGN_CERTIFICATE_FINGERPRINT: ${$NUGET_SIGN_CERTIFICATE_FINGERPRINT}
```

Comment:
is ${$NUGET_SIGN_CERTIFICATE_FINGERPRINT} right?
and not `${NUGET_SIGN_CERTIFICATE_FINGERPRINT}`

Reply:
Good catch! Thank you!